### PR TITLE
Update dns-ownership documentation.

### PR DESCRIPTION
**Pull request by 1marcghannam**
1. Adapter has been updated: Passing type as TXT is no longer needed.
2. Ethereum job cost changed from 1 LINK to 2 LINK.

### CODE REVIEW

**Comment by aelmanaa** [comment]
hello @1marcghannam 

thanks so much for the PR. Indeed I'm able to get a response using the fixes that you suggested.
Just for my understanding: How do you map an IP address to an Ethereum address ? 

This is the response I get from `https://dns.google/resolve?name=www5.infernos.io`:

```json
{
  "Status": 0,
  "TC": false,
  "RD": true,
  "RA": true,
  "AD": false,
  "CD": false,
  "Question": [
    {
      "name": "www5.infernos.io.",
      "type": 1
    }
  ],
  "Answer": [
    {
      "name": "www5.infernos.io.",
      "type": 1,
      "TTL": 300,
      "data": "172.67.143.249"
    },
    {
      "name": "www5.infernos.io.",
      "type": 1,
      "TTL": 300,
      "data": "104.21.71.82"
    }
  ],
  "Comment": "Response from 108.162.193.195."
}
```


Additional info: similar PR still in review https://github.com/smartcontractkit/external-adapters-js/pull/1658

**Comment by 1marcghannam** [comment]
Hello @aelmanaa ,

I appreciate the review. Your query is missing `type` parameter which is always set to `TXT`.
The correct query should be `https://dns.google/resolve?name=www5.infernos.io&type=TXT` with response:

```json
{
    "Status": 0,
    "TC": false,
    "RD": true,
    "RA": true,
    "AD": false,
    "CD": false,
    "Question": [
        {
            "name": "www5.infernos.io.",
            "type": 16
        }
    ],
    "Answer": [
        {
            "name": "www5.infernos.io.",
            "type": 16,
            "TTL": 300,
            "data": "0x2ffb3d72fa6af12af5d378df0697a2bf9f45652e"
        },
        {
            "name": "www5.infernos.io.",
            "type": 16,
            "TTL": 300,
            "data": "0x4d3407ddfdeb3feb4e8a167484701aced7056826"
        },
        {
            "name": "www5.infernos.io.",
            "type": 16,
            "TTL": 300,
            "data": "0xeb9abc589734ce8051f089cf3498e230456a85dd"
        },
        {
            "name": "www5.infernos.io.",
            "type": 16,
            "TTL": 300,
            "data": "0xf75519f611776c22275474151a04183665b7feDe"
        }
    ],
    "Comment": "Response from 108.162.192.117."
}
```

**Comment by aelmanaa** [comment]
> Hello @aelmanaa ,
> 
> I appreciate the review. Your query is missing `type` parameter which is always set to `TXT`. The correct query should be `https://dns.google/resolve?name=www5.infernos.io&type=TXT` with response:
> 
> ```json
> {
>     "Status": 0,
>     "TC": false,
>     "RD": true,
>     "RA": true,
>     "AD": false,
>     "CD": false,
>     "Question": [
>         {
>             "name": "www5.infernos.io.",
>             "type": 16
>         }
>     ],
>     "Answer": [
>         {
>             "name": "www5.infernos.io.",
>             "type": 16,
>             "TTL": 300,
>             "data": "0x2ffb3d72fa6af12af5d378df0697a2bf9f45652e"
>         },
>         {
>             "name": "www5.infernos.io.",
>             "type": 16,
>             "TTL": 300,
>             "data": "0x4d3407ddfdeb3feb4e8a167484701aced7056826"
>         },
>         {
>             "name": "www5.infernos.io.",
>             "type": 16,
>             "TTL": 300,
>             "data": "0xeb9abc589734ce8051f089cf3498e230456a85dd"
>         },
>         {
>             "name": "www5.infernos.io.",
>             "type": 16,
>             "TTL": 300,
>             "data": "0xf75519f611776c22275474151a04183665b7feDe"
>         }
>     ],
>     "Comment": "Response from 108.162.192.117."
> }
> ```

Thanks for the info @1marcghannam ! 
Just made a few updates:
- Update the solidity contract accordingly
- Write a guide so it is easy for users to test in one-click
- Fixed some broken links

Is your node still using V1? The _Tasks_ section mentions V1 (e.g.: https://docs.chain.link/docs/core-adapters/#copy). If you are node operates on V2 then we should update the section accordingly, mentioning the right [V2 tasks](https://docs.chain.link/docs/tasks/).

**Comment by 1marcghannam** [comment]
> > Hello @aelmanaa ,
> > I appreciate the review. Your query is missing `type` parameter which is always set to `TXT`. The correct query should be `https://dns.google/resolve?name=www5.infernos.io&type=TXT` with response:
> > ```json
> > {
> >     "Status": 0,
> >     "TC": false,
> >     "RD": true,
> >     "RA": true,
> >     "AD": false,
> >     "CD": false,
> >     "Question": [
> >         {
> >             "name": "www5.infernos.io.",
> >             "type": 16
> >         }
> >     ],
> >     "Answer": [
> >         {
> >             "name": "www5.infernos.io.",
> >             "type": 16,
> >             "TTL": 300,
> >             "data": "0x2ffb3d72fa6af12af5d378df0697a2bf9f45652e"
> >         },
> >         {
> >             "name": "www5.infernos.io.",
> >             "type": 16,
> >             "TTL": 300,
> >             "data": "0x4d3407ddfdeb3feb4e8a167484701aced7056826"
> >         },
> >         {
> >             "name": "www5.infernos.io.",
> >             "type": 16,
> >             "TTL": 300,
> >             "data": "0xeb9abc589734ce8051f089cf3498e230456a85dd"
> >         },
> >         {
> >             "name": "www5.infernos.io.",
> >             "type": 16,
> >             "TTL": 300,
> >             "data": "0xf75519f611776c22275474151a04183665b7feDe"
> >         }
> >     ],
> >     "Comment": "Response from 108.162.192.117."
> > }
> > ```
> 
> Thanks for the info @1marcghannam ! Just made a few updates:
> 
> * Updated the solidity contract accordingly
> * Wrote a guide so it is easy for users to test in one-click
> * Fixed some broken links
> 
> Is your node still using V1? The _Tasks_ section mentions V1 (e.g.: https://docs.chain.link/docs/core-adapters/#copy). If you are node operates on V2 then we should update the section accordingly, mentioning the right [V2 tasks](https://docs.chain.link/docs/tasks/).

Thank you for the updates @aelmanaa ,
We are using V2 tasks in the updated jobs.

```TOML
type = "directrequest"
schemaVersion = 1
contractAddress = "0x0000000000000000000000000000000000000000"
maxTaskDuration = "0s"
observationSource = """
    decode_log          [type=ethabidecodelog
                         abi="OracleRequest(bytes32 indexed specId, address requester, bytes32 requestId, uint256 payment, address callbackAddr, bytes4 callbackFunctionId, uint256 cancelExpiration, uint256 dataVersion, bytes data)"
                         data="$(jobRun.logData)"
                         topics="$(jobRun.logTopics)"]

    decode_cbor         [type=cborparse data="$(decode_log.data)"]

    dnsproof            [type=bridge
                         name="dnsproof"
                         requestData="{\\"data\\": {\\"endpoint\\": \\"dnsProof\\", \\"name\\": $(decode_cbor.name), \\"record\\": $(decode_cbor.record)}}"]
                         

    result_parse        [type=jsonparse data="$(dnsproof)" path="result"]

    encode_data         [type=ethabiencode
                         abi="(bool _result)"
                         data="{\\"_requestId\\": $(decode_log.requestId),\\"_result\\": $(result_parse)}"]

    encode_tx           [type=ethabiencode
                         abi="fulfillOracleRequest(bytes32 requestId, uint256 payment, address callbackAddress, bytes4 callbackFunctionId, uint256 expiration, bytes32 data)"
                         data="{\\"requestId\\": $(decode_log.requestId),\\"payment\\": $(decode_log.payment),\\"callbackAddress\\": $(decode_log.callbackAddr),\\"callbackFunctionId\\": $(decode_log.callbackFunctionId),\\"expiration\\": $(decode_log.cancelExpiration),\\"data\\": $(encode_data)}"]

    submit_tx           [type=ethtx to="0x0000000000000000000000000000000000000000" data="$(encode_tx)" minConfirmations="2"]

    decode_log -> decode_cbor -> dnsproof -> result_parse -> encode_data -> encode_tx -> submit_tx
"""

```

**Comment by aelmanaa** [comment]
@1marcghannam  you can check the new page here:
https://documentation-git-fork-1marcghannam-patch-1-chainlinklabs.vercel.app/docs/dns-ownership-oracle#dns-ownership-contract

Are you happy with the result? If yes then will merge so it is live

File: _includes/samples/DataProviders/DnsOwnership.sol [hunk 1]
```diff
@@ -13,42 +13,49 @@ contract DnsOwnershipChainlink is ChainlinkClient, ConfirmedOwner {
...
      *      Name:           DNS Record Check
      *      Listing URL:    https://market.link/jobs/bf1a410f-ce93-497d-83ac-e63fed9d83bd?network=42
      *      ID:             791bd73c8a1349859f09b1cb87304f71
      *      Fee:            0.1 LINK
      */
-    constructor(uint256 _oraclePayment) ConfirmedOwner(msg.sender) {
+    constructor() ConfirmedOwner(msg.sender) {
       setPublicChainlinkToken();
-      oraclePayment = _oraclePayment;
+      oracle = 0xfF07C97631Ff3bAb5e5e5660Cdf47AdEd8D4d4Fd;
```
Comment (pappas999) [inline _includes/samples/DataProviders/DnsOwnership.sol:33]: is this oracle address correct? looking at the job on [market.link](https://market.link/jobs/bf1a410f-ce93-497d-83ac-e63fed9d83bd) it has another listed oracle contract

**Review by pappas999 — COMMENTED** [review]
looks good other than my 1 comment

File: _includes/samples/DataProviders/DnsOwnership.sol [hunk 2]
```diff
@@ -13,42 +13,49 @@ contract DnsOwnershipChainlink is ChainlinkClient, ConfirmedOwner {
...
+    address private oracle;
+    bytes32 private jobId;
     bool public proof;
     /**
      * Network: Kovan
      * Oracle: 
      *      Name:           LinkPool
      *      Listing URL:    https://market.link/nodes/323602b9-3831-4f8d-a66b-3fb7531649eb?network=42
-     *      Address:        0x56dd6586DB0D08c6Ce7B2f2805af28616E082455
+     *      Address:        0xff07c97631ff3bab5e5e5660cdf47aded8d4d4fd
```
Comment (dwightjl) [inline _includes/samples/DataProviders/DnsOwnership.sol:24]:
```suggestion
     *      Address:        0xfF07C97631Ff3bAb5e5e5660Cdf47AdEd8D4d4Fd
```

Remix didn't like the case on this address. Updated to match.

**Review by dwightjl — CHANGES_REQUESTED** [review]
Tested this and it works. Looks good with just a few suggested edits. Applying them now.

**Review by dwightjl — APPROVED** [review]

**Pull request merged**